charts/zeebe-benchmark/README.md

```diff
@@ -80,9 +80,20 @@ Camunda kubernetes deployment (such as in Google cloud, AWS etc.), you
 need to run through the following steps:
 
 1. Create a new API client in the intended cluster.
-2. From the environment vars in the newly created client we need to copy 
+2. Download environment vars Under "Env Vars" tab
+3. From the environment vars in the newly created client we need to copy
    them to our `values.yaml` under `saas.credentials` [here](https://github.com/camunda/zeebe-benchmark-helm/blob/751c7b35e8041da9c5e7a75232cb4d43942e6ac3/charts/zeebe-benchmark/values.yaml#L45-L78).
-3. Set `saas.enabled` to true. 
+   Set `saas.enabled` to true.
+   This can be done with a script
+
+    ```sh
+    # export env vars
+    # eg source ~/Downloads/CamundaCloudMgmtAPI-Client-download-test.txt
+
+    # require yq
+    ./scripts/update-values-saas-from-env.sh
+    ```
```

Comment:
In my opinion it would be simpler to just have an example like this here:

```
$ helm install ck-gameday zeebe-benchmark/zeebe-benchmark \
  --set camunda-platform.enabled=false \
  --set saas.enabled=true \
  --set saas.credentials.clientId="$ZEEBE_CLIENT_ID" \
  --set saas.credentials.clientSecret="$ZEEBE_CLIENT_SECRET" \
  --set saas.credentials.zeebeAddress="$ZEEBE_ADDRESS" \
  --set saas.credentials.authServer="$ZEEBE_AUTHORIZATION_SERVER_URL"
```

I don't think we need to overwrite the local values file, especially if you simply want to run the benchmarks you don't necessarily need to clone the github repo.